### ISSUE TITLE
Reprint!

### ISSUE THREAD

**Issue by chadwhitacre** [gospeldesk/the-gospels#1]
- [x] find original caslon.ttf
- [x] set up title on IngramSpark!
- [x] understand [file requirements](https://www.ingramcontent.com/publishers/resources/digital-bookblock)
- [x] replace title & copyright pages
- [x] drop inside gutter (final size 6.125 x 9.25)
- [x] run through Acrobat
- [x] design a cover: #2
- [x] matte or gloss? matte (per Nick, but I think we can get both from IngramSpark to compare?)
- [x] publication/on-sale date? Let's aim for ~May 26~ June 30.
- [x] ~[request samples](https://help.ingramspark.com/hc/en-us/articles/115001409963-Request-Sample-Books-to-See-Print-Quality)~—too late, I had thought this was samples of _my_ book

**Comment by chadwhitacre** [comment]
Found Caslon! 💃 😍 🚀 ✨

**Comment by chadwhitacre** [comment]
Dusting off the Makefile with modern Prince:

```
$ make
cat webhtm/Mark.htm \
                webhtm/Matthew.htm \
                webhtm/Luke.htm \
                webhtm/John.htm \
                > gospels.raw.htm
python cleanup.py
801 apostrophes saved from certain death
31 more apostrophes saved from certain death
29 sections added
cat preface.htm gospels.htm > all.htm
./prince all.htm \
                --style=gospels.css \
                --output=gospels.prince.pdf \
                --no-default-style
prince: gospels.css:191: warning: unknown property 'ackground'
prince: gospels.css:192: warning: unknown property 'ackground'
prince: gospels.css:204: warning: unknown property 'ont-variant'
prince: gospels.css:208: warning: unknown property 'ounter-increment'
prince: gospels.css:213: warning: unknown property 'ackground'
prince: gospels.css:221: warning: unknown property 'ackground'
prince: gospels.css:280: warning: unknown property 'order-bottom'
prince: gospels.css:286: warning: unknown property 'order-bottom'
prince: gospels.css:294: warning: unknown property 'ackground'
prince: gospels.css:299: warning: unknown property 'page-break'
prince: gospels.css:306: warning: unknown property 'ackground'
prince: gospels.css:310: warning: unknown property 'ackground'
prince: gospels.css:317: warning: unknown property 'ackground'
$
```

**Comment by chadwhitacre** [comment]
First push in eight years! 4b72772a6eaf5914cd1d37b7c13718ec54c65d76 😂

**Comment by chadwhitacre** [comment]
https://www.youtube.com/watch?v=dBbnMfKVAGA

**Comment by chadwhitacre** [comment]
> Use single-page format (1-up per page)

https://help.ingramspark.com/hc/en-us/articles/211151743-Creating-and-Designing-Print-Ready-Digital-Files-for-Your-Book

Yesssssssss. 👍 That's the format I have the v1.2.1 file in. Can I get away with just replacing the title and copyright pages? 🤞That'll save a lot of effort trying to redo with Prince 13.

**Comment by chadwhitacre** [comment]
I found a way to crop off the inside gutter, using [Acrobat JavaScript](https://www.adobe.com/content/dam/acom/en/devnet/acrobat/pdfs/js_api_reference.pdf) (see the [PDF spec](https://www.adobe.com/content/dam/acom/en/devnet/pdf/pdfs/PDF32000_2008.pdf) for documentation on `MediaBox`, etc.):

1. Open [Acrobat Pro](https://acrobat.adobe.com/us/en/acrobat/acrobat-pro.html) (7-day trial! 💃).
1. Open the PDF file to be cropped.
1. Run some JavaScript to crop down from 450 (6.25") to 441 points (6.125") wide, alternating left/right crop on even/odd pages:
    1. Hit ⌘-J to open the JavaScript console.
    1. Paste this in the console view area: `for (var i=0; i < this.numPages; i++) { this.setPageBoxes('Media', i, i, i%2 ? [0,666,441,0] : [9,666,450,0]); }`.
    1. Hit ⌘-Enter at the end of the line to execute it (output appears in the same textarea[!]).

Then, here's the method I'm using to produce a file in press-ready PDF/X-3:2002 format:

1. File > Export to Postscript > Save (default settings are fine)
1. Open Acrobat Distiller.
1. Default Settings: PDF/X-3:2002.
1. ⌘-O and open the exported *.ps file from step (1).

Voilà! Distiller took ~30 seconds for me, ended up with a cropped file in press-ready format. 💃 I took screenshots of before and after to convince myself (below). Now to swap in new title and copyright pages!

# Before

![before](https://user-images.githubusercontent.com/134455/77238468-9c585800-6ba6-11ea-92e0-b2f3cf33d0bb.png)

# After

![after](https://user-images.githubusercontent.com/134455/77238469-9c585800-6ba6-11ea-9f05-a579140d0029.png)

**Comment by chadwhitacre** [comment]
New title and copyright page in 60aa8e1ec1150039d2199607b5c855c336b159e5.

Gonna hand-roll the final PDF to upload. Then I need a cover!

**Comment by chadwhitacre** [comment]
Done. I copied `The Gospels (v1.2.1).pdf` to `The Gospels (v1.2.2).pdf`, then used Adobe Acrobat to replace the title and copyright page with the new pages from `title.pdf`. Then I picked up with https://github.com/gospel-desk/the-gospels/issues/1#issuecomment-602117784. Here it is!

[The Gospels (v1.2.2).pdf](https://github.com/gospel-desk/the-gospels/files/4364602/The.Gospels.v1.2.2.pdf)

**Comment by chadwhitacre** [comment]
Alright, tinkered with the title/copyright page and today is the last day before Adobe bills me. Gonna hand-roll again:

- [ ] Start with [Gospels 1.2.1.pdf](https://github.com/gospel-desk/the-gospels/files/4392665/The.Gospels.v1.2.1.pdf) from last time.
- [ ] Use Acrobat to replace the title and copyright pages.
- [ ] Use Acrobat to fix a typo in the acknowledgements (I spelled Schroth's name wrong :( ).
- [ ] Trim off the inside gutter and generate a press-ready PDF: https://github.com/gospel-desk/the-gospels/issues/1#issuecomment-602117784.

**Comment by chadwhitacre** [comment]
Title page in 0c8c23c2eee66545e1ee818e269b8940593490fe.

**Comment by chadwhitacre** [comment]
[The Gospels (v1.2.2).ps.zip](https://github.com/gospel-desk/the-gospels/files/4392734/The.Gospels.v1.2.2.ps.zip)

**Comment by chadwhitacre** [comment]
[The Gospels (v1.2.2).pdf](https://github.com/gospel-desk/the-gospels/files/4393102/The.Gospels.v1.2.2.pdf)

https://github.com/gospel-desk/www.gospel-desk.com/commit/d833e810f85ac3eafdb5bb944d7a37a63cd2ac90

https://www.gospel-desk.com/the-gospels/The%20Gospels%20(v1.2.1).pdf

**Comment by chadwhitacre** [comment]
> Next payment is scheduled for Mar 29, 2020.

**Comment by chadwhitacre** [comment]
![Screen Shot 2020-03-27 at 7 31 31 AM](https://user-images.githubusercontent.com/134455/77751951-06d12400-6ffd-11ea-844e-a3e033006022.png)

**Comment by chadwhitacre** [comment]
> You cancelled your Acrobat Pro DC subscription (month-to-month). Your account will continue to work until the end of your billing cycle on 27-March-2020 (PT), but after this date you will lose access to the features and services of your subscription.

**Comment by chadwhitacre** [comment]
💃 

![Screen Shot 2020-06-11 at 8 40 50 AM](https://user-images.githubusercontent.com/134455/84386402-48ff1000-abbf-11ea-80ac-8160309b7b98.png)

**Comment by chadwhitacre** [comment]
I had to have Nick redo for 242 pages vs. 240 to make room for a bio page at the back. Link above updated.

**Comment by chadwhitacre** [comment]
Title/copyright/bio revisions in 650dee6c0ce26530b559c44df9b9603adbe1c291.

**Comment by chadwhitacre** [comment]
Preface edits in d8a69c4d15fbb174f2adf7b8e62cb45b27feaad0.

**Comment by chadwhitacre** [comment]
To hand-roll again:

- [ ] Start with [Gospels 1.2.1.pdf](https://github.com/gospel-desk/the-gospels/files/4392665/The.Gospels.v1.2.1.pdf) from last time.
- [ ] Use Acrobat to replace the title, copyright, and two preface pages.
- [ ] Use Acrobat to modify the title and author metadata.
- [ ] Trim off the inside gutter: https://github.com/gospel-desk/the-gospels/issues/1#issuecomment-602117784.
- [ ] Export to Postscript.
- [ ] Rip whit537 out of the *.ps.
- [ ] Distiller to press-ready PDF.

**Comment by chadwhitacre** [comment]
![c8vhGpDVcS](https://user-images.githubusercontent.com/134455/84658405-dc984f80-aee3-11ea-8106-369e4700ec95.gif)

**Comment by chadwhitacre** [comment]
[The Gospels (v1.2.3).pdf](https://github.com/gospel-desk/the-gospels/files/4780305/The.Gospels.v1.2.3.pdf)

**Comment by chadwhitacre** [comment]
![Screen Shot 2020-06-15 at 8 54 23 AM](https://user-images.githubusercontent.com/134455/84659708-d60ad780-aee5-11ea-991e-214a1d2b7fad.png)

---

> ## Congratulations! Your title submission was successful.
>
> For all file uploads: Please allow 1-2 business days for technical file review. Please note that even though your title has passed initial validation, there may be additional errors that can only be detected through a visual inspection. Note: this does not include editorial or design issues unless they directly prevent your book's ability to be printed. If this is the case, you will be notified by email.
>
> Proofing process: Following visual inspection, an electronic proof of your print title will be provided for your review, and you will be notified via email when it is ready. You must approve your eproof before your print title can be printed or sold.

**Comment by chadwhitacre** [comment]
Be still, my beating heart.

**Comment by chadwhitacre** [comment]
$49

**Comment by chadwhitacre** [comment]
Shoot. Looks like I was supposed to account for the page they add at the end in the cover template. Should be 244 not 242. Will wait for their review, I guess?

![Screen Shot 2020-06-15 at 9 14 33 AM](https://user-images.githubusercontent.com/134455/84661644-a4474000-aee8-11ea-99ac-c73812b59b7e.png)

**Comment by chadwhitacre** [comment]
No news. Maybe I can upload w/ 241 pages and they'll add just the final verso? That would save another round-trip with Nick, but will probably still cost another $25.

**Comment by chadwhitacre** [comment]
E-proof! They got the page count right! 💃 

[EPROOF-098515490X.pdf](https://github.com/gospel-desk/the-gospels/files/4794098/EPROOF-098515490X.pdf)

**Comment by chadwhitacre** [comment]
So much for June 30. :-/

![Screen Shot 2020-06-17 at 1 03 53 PM](https://user-images.githubusercontent.com/134455/84927693-30996480-b09b-11ea-8877-93cbcb1d696a.png)

**Comment by chadwhitacre** [comment]
> Thank you. Your order has been submitted for processing.

~$11 (shipping tbd)

**Comment by chadwhitacre** [comment]
I made these variants in case I needed them after Acrobat trial expired.

[gospels-1.2.3-240.pdf](https://github.com/gospel-desk/the-gospels/files/4794134/gospels-1.2.3-240.pdf)
[gospels-1.2.3-241.pdf](https://github.com/gospel-desk/the-gospels/files/4794135/gospels-1.2.3-241.pdf)
[gospels-1.2.3-242.pdf](https://github.com/gospel-desk/the-gospels/files/4794133/gospels-1.2.3-242.pdf)

**Comment by chadwhitacre** [comment]
Sooooo ... now I wait a month. 🐭

**Comment by chadwhitacre** [comment]
I just "enable[d] distribution" on IngramSpark. There was no fee associated so why not? I want to see what it looks like to buy this thing on Amazon! Does it take four weeks to fulfill? 😩

**Comment by chadwhitacre** [comment]
Hrm, I had put the "for sale" date at June 30, so not sure it'll be orderable until then anyway. But maybe it'll get listed as "coming soon" or whatevs?

**Comment by chadwhitacre** [comment]
> Due to extremely high volume and our commitment to protecting the health and safety of our associates, U.S. print turnaround times are longer than normal. Our best estimate for each day is available at the time you place your order. Please know the posted estimate is subject to change – positively or negatively – from the time you place your order as we work to return to our standard turnaround times.

**Comment by chadwhitacre** [comment]
> We are only able to submit a request to Amazon.com, Amazon.ca, and Amazon.co.uk once a title has been in distribution for at least 30 days.

https://www.ingramspark.com/blog/ingramspark-title-availability

**Comment by chadwhitacre** [comment]
Moreover:

> Amazon.com is one of our most popular retail channels, and we are able to submit a request to Amazon directly concerning your title’s availability, should the need arise. If the title is listed as completely unavailable or “temporarily out of stock,” we request they update your title's availability status. How Amazon lists a title and the availability status on their website is ultimately up to them. Amazon's purchasing decisions are supported by an algorithm that is proprietary to Amazon. In choosing to stock titles or even show products as available on their sites, Amazon considers many pieces of information about your book. Factors in a decision to stock a book may include discount (with a maximum trade discount being most favorable), whether a book is returnable, product page views, and sales history. Even print-on-demand books may not show an "in stock" message at Amazon.

Hence #4.

**Comment by chadwhitacre** [comment]
Closing in favor of #5.

**Comment by chadwhitacre** [comment]
> Under normal circumstances, a title should be available for purchase on most websites within 4 weeks from when the distribution service was enabled/added by you. Please note that if this is the first title you set up on the IngramSpark platform, this time can extend up to 6-8 weeks.

https://help.ingramspark.com/hc/en-us/articles/209072566-Title-Listing-on-Websites

😔

**Comment by chadwhitacre** [comment]
It arrived! The quality is too low, both the print quality and the distribution quality. The paper is too thin and the book isn't cut right. For distribution I ordered on Amazon on July 6, the first day I noticed it. It said 2-day prime (and I had seen a wholesale order come through on IngramSpark). But now my order is saying "Expected August 26." 😩 

![IMG-3871](https://user-images.githubusercontent.com/134455/87860492-deeb3080-c90b-11ea-84cb-958ff8498faa.jpg)

![IMG-3870](https://user-images.githubusercontent.com/134455/87860493-deeb3080-c90b-11ea-85f5-e5bcfc5fbe47.JPG)

![IMG-3869](https://user-images.githubusercontent.com/134455/87860494-deeb3080-c90b-11ea-8ad7-c07a766c1eec.JPG)
